.github/CODEOWNERS

```diff
@@ -11,7 +11,7 @@
 #   3. Use the command palette to run the CODEOWNERS: Show owners of current file command, which will display all code owners for the current file.
 
 # Default ownership for all repo files
-* @anasalkouz @andrross @ashking94 @Bukhtawar @CEHENKLE @dblock @dbwiddis @gbbafna @kotwanikunal @linuxpi @mch2 @msfroh @nknize @owaiskazi19  @reta @Rishikesh1159 @sachinpkale @saratvemulapalli @shwetathareja @sohami @VachaShah
+* @anasalkouz @andrross @ashking94 @Bukhtawar @CEHENKLE @dblock @dbwiddis @gbbafna @jainankitk @kotwanikunal @linuxpi @mch2 @msfroh @nknize @owaiskazi19  @reta @Rishikesh1159 @sachinpkale @saratvemulapalli @shwetathareja @sohami @VachaShah
```

Comment:
I noticed that for some of the new maintainers we have been adding them to just this line, some to only the `/MAINTAINERS.md` below (L37) and some to both. Should we try to standardize this process if that's not already written down somewhere?

Reply:
I think this is the default, but the maintainers can choose to mark themselves as codeowners of sub-folders?

Let's resolve this one and merge the PR, do bring this issue up if you think something is to be done!